combat/CombatLogManager.cpp

```diff
@@ -64,18 +64,10 @@ namespace {
     };
 }
 
-CombatParticipantState::CombatParticipantState():
-current_health(0.0f),
-max_health(0.0f)
-{
-
-}
-
+CombatParticipantState::CombatParticipantState()
+{}
 
-CombatParticipantState::CombatParticipantState(const UniverseObject& object):
-current_health(0.0f),
-max_health(0.0f)
-{
+CombatParticipantState::CombatParticipantState(const UniverseObject& object) {
     FillState(*this, object);
```

Comment:
I thought you meant line 67 for the call with the empty argument list.
That used to have {} on a new line (compared to before commit 	c36b834 ).

If you are referring to the function that uses "object" as an argument, then it shouldn't have {} on one line because the function body is not empty: `FillState(*this, object);`.